sky/cli.py

```diff
@@ -1540,6 +1542,8 @@ def autostop(
 
     ``--idle-minutes`` is the number of minutes of idleness (no pending/running
```

Comment:
Proposal: add a paragraph before this: "If ``--down`` is passed, autodown (tear down the cluster; non-restartable) is used, rather than autostop (restartable)."

Consider also changing the first line? "Schedule or cancel an auto-stop or auto-down for cluster(s)."